java/ql/lib/semmle/code/java/security/UnsafeDeserializationQuery.qll

```diff
@@ -300,6 +267,70 @@ class UnsafeDeserializationConfig extends TaintTracking::Configuration {
   }
 }
 
+/** Taint step for Unsafe deserialization */
+private class UnsafeDeserializationAdditionalTaintStep extends Unit {
+  predicate isAdditionalTaintStep(DataFlow::Node pred, DataFlow::Node succ) {
```

Comment:
No need for this to be a member predicate on a unit type - that's only a needed trick for making a binary (or greater arity) extension point. Since this is private we can just make it a simple top-level predicate.